mobify/platform-scaffold
# Prepare for breaking SDK changes

This PR adds the dependencies necessary to be compatible with the breaking changes in the upcoming SDK release. It can be merged before upgrading to the new SDK because these additional dependencies cause no problems with the current SDK release.

 **JIRA**: N/A

## Changes
- Explicitly depend on `react-addons-shallow-compare` and `react-addons-transition-group` in the web directory.

## How to test-drive this PR
- npm i
- see that things still work
- link in `develop` of the SDK
- npm i
- see that things still work
